.Rprofile

```diff
@@ -1,5 +1,17 @@
 source("renv/activate.R")
 
+options(repos = c(pikruniverse = "https://pik-piam.r-universe.dev", CRAN = "https://cran.rstudio.com/"))
+
+# bootstrapping, will only run once after remind is freshly cloned
+if (identical(rownames(installed.packages(priority = "NA")), c(Package = "renv")) &&
```

Comment:
Just wondering: what kind of error does that throw if the RSE package repository is not configured correctly?  Maybe capture that and give an explicit hint on how to do that.